db/migrate/20221020123234_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly, Do the same as requested in the previous comment.

Reply:
Hello, @RolandM99
- Thanks for the heads up, the explanation is clear